optimum/amd/brevitas/configuration.py

```diff
@@ -54,7 +54,9 @@ class BrevitasQuantizationConfig:
             - `"layerwise"`: Apply SmoothQuant as described in https://arxiv.org/abs/2211.10438. The activation rescaling will be added as multiplication node, that is not fused within a preceding layer.
             - `"cross_layer"`: Apply SmoothQuant, and fuse the activation rescaling within a preceding layer when possible (example: nn.LayerNorm followed by nn.Linear). This is achieved through a graph capture of the model using [torch.fx](https://pytorch.org/docs/stable/fx.html#module-torch.fx).
         apply_weight_equalization (`bool`, defaults to `False`):
-            Applies weight equalization accross layers, following https://arxiv.org/abs/1906.04721. This parameter is useful for models whose activation function is linear or piecewise-linear (like ReLU, used in OPT model), and allows to reduce the quantization error of the weights by balancing scales accross layers.
+            Applies weight equalization across layers, following https://arxiv.org/abs/1906.04721. This parameter is useful for models whose activation function is linear or piecewise-linear (like ReLU, used in OPT model), and allows to reduce the quantization error of the weights by balancing scales across layers.
+        apply_bias_correction (`bool`, defaults to `False`):
+            Applies bias correction to compensate for changes in activation bias caused by quantization
```

Comment:
```suggestion
            Applies bias correction to compensate for changes in activation bias caused by quantization.
```

Can you also link to a reference?

Reply:
The implementation is loosely based on the concepts in the weight equalisation [paper](https://arxiv.org/abs/1906.04721), but we use calibration data to generate the statistics, instead of relying on batchnorm + other linear transforms to estimate them. Not sure we have a specific reference for that, but I think other works are using a similar approach...